### ISSUE TITLE
After update 2.19.6 cannot access admin panel

### ISSUE THREAD

**Issue by Reis-A** [the-djmaze/snappymail#602]
**Describe the bug**
snappy mail tries to login the Nextcloud admin account into the admin panel automatically.
however, in our setup, the admin account and its password is a separate account from Nextcloud admin.
historically,  comes from the rain loop admin account we had.
when i enter the correct login data, i get the error message no connection to server.
the good thing is, the mail system still is still working for the users.

**To Reproduce**
1. got to..

**Expected behavior**

**Screenshots**

**Please complete the following information:**
- Browser: [e.g. chromium 100, firefox 100, safari 14, mobile] safari
- IMAP daemon: [e.g. courier, dovecot] groupwise
- PHP version: 8.1.9
- SnappyMail Version: 2.19.6

**[Debug/logging information](https://github.com/the-djmaze/snappymail/wiki/FAQ#how-do-i-enable-logging)**
Place them here (few lines) or as attachments (many lines)

**Additional context**

**Comment by the-djmaze** [comment]
I'm no Nextcloud expert, just trying along the way.
So you have something like:
1. Admin account created at install of Nextcloud
2. Second account and gave it admin rights
3. Trying to use SnappyMail ?admin with second account?

**Comment by HeyVern** [comment]
Can confirm serveral issues with 2.19.6.  Admin panel does not load, and when logged in Lists are Empty (not showing any mail).  Rolled back to 2.19.4 and everything was fine again.

**Comment by Reis-A** [comment]
my setup is like this:
1. local admin account on nextcloud was created at installation of
nextcloud, username: xyadmin.
(xyadmin is not an ldap account and has no imap-emailadress, (could be
setup if needed)
2. rainloop app was installed with xyadmin. the default admin account in
rainloop had username admin, i changed the default password
3. snappymail was installed, admin credentials were taken from rainloop.
rainloop ist now deactivated on nextcloud.
4. snappy mail 2.19.6 was installed. now when i login with
xyadmin snappymail tries to connect, but fails. the username is then filled
out with the nextcloud admin username xyadmin. when i change it to the
rainloop credentials "admin" with rainloop password, i get " no connection
to server"

**Comment by the-djmaze** [comment]
It seems you have the wrong login screen.

Admin panel is now integrated/embed as well so make sure the url is
`/index.php/apps/snappymail/?admin`

If that fails try the old way
`/index.php/apps/snappymail/run/?admin`

**Comment by sgutermann** [comment]
Complete fresh install of Nextcloud v25 and SnappyMail 2.19.6. 

I can not get into the admin panel, say auth is wrong.

Repeated that from scratch twice.

**Comment by HeyVern** [comment]
@the-djmaze the old link works to get to admin panel, but the old link is not what's being refered to on the Additional Settings page.  However, the other issue, of mail not showing up in the Inbox still persists.  The count shows the appropriate number, but the list is empty.  Seems like there are a few issues with 2.19.6.

**Comment by the-djmaze** [comment]
> Seems like there are a few issues with 2.19.6.

True, every time i try to integrate/embed it more and more to a good experience.

There are so many different Nextcloud setups and if mine works it doesn't mean everyone else does.

But eventually we get there :smile:

**Comment by the-djmaze** [comment]
So here's mine that works so you see it looks better :wink:
![afbeelding](https://user-images.githubusercontent.com/3752035/199304661-fcd17453-fa7d-4320-a71b-fbfa3cdbccc8.png)

**Comment by sgutermann** [comment]
Ok. If I click on the BETA IFrame thing I can get in. Via the normal page I can not.

Once I configured my domain (mailserver name, starttls, etc.) I can log in to the Mail ... all my Mail folders do apear, but there is no mail.

@the-djmaze - when you say your setup - I am not sure I can follow. I use a fresh debian 11 server, install apache2 and mysql. Download thenextcloud 25 package, unpack and get started.

What are you doing that is differen oder what part of the installation or differences ar you referring to?

In other words, what can I do to help?

**Comment by the-djmaze** [comment]
> In other words, what can I do to help?

You already did!
It's the embed mode that is the issue.

Shall i just keep it embed and remove the other and out of beta then?

**Comment by sgutermann** [comment]
I personally prefer the embed mode. Looks way better. However I just noticed that I do NOT get to see my mails.

I get to see the Folders, etc. and it looks like my in-box, but the actual Mail does not seem to be displayed.

I will wait a few minutes ... but doe not look good just now.
